README.md

```diff
@@ -195,6 +195,22 @@ the entire value be managed as a secured secret.
 
 The user must have the `readWrite` role or a custom role with specific privileges.
 
+## MESH API
+
+TODO: NHAIS Adaptor MESH configuration when MESH integration is implemented
+
+For local test scripts see [mesh/README.md](/mesh/README.md)
+
+### OpenTest
+
+TODO
+
+### Fake MESH
+
+Clone https://github.com/mattd-kainos/fake-mesh from our fork
```

Comment:
Why not in this repo?

Reply:
Different project, different tech stack, and it is a fork of someone else's repository. Better to leave separate.